omnigibson/systems/micro_particle_system.py

```diff
@@ -497,7 +450,7 @@ def use_isosurface(cls):
                 See https://docs.omniverse.nvidia.com/app_create/prod_extensions/ext_physics.html?highlight=isosurface#isosurface
                 for more information
         """
-        raise NotImplementedError()
+        return False
```

Comment:
why are we returning false by default for all of these properties?

Reply:
Because we have been setting them as False always? This is just to remove some redundancy. As far as I understand, only the fluid system set isosurface to be True.

Reply:
gotcha. makes sense!